### ISSUE TITLE
Runs on projects if there is a parent directory called "templates" 

### ISSUE THREAD

**Issue by pbnj** [tim-koehler/Helm-Intellisense#29]
### Problem

I have a project in a path which includes `templates` in the directory chain. On every save, I get a `could not locate any values.yaml` error.

### Reproduction Steps

```shell
$ mkdir -p /tmp/templates/foo
$ code /tmp/templates/foo
# create a README.md and save, you should get an error
```

```shell
$ mkdir -p /tmp/bar
$ code /tmp/bar
# create a README.md and save, you should NOT get an error
```

### Screenshot

![Screen Shot 2021-02-22 at 3 21 48 PM](https://user-images.githubusercontent.com/6811830/108783110-b078a800-7521-11eb-947f-05503b3c9521.png)

**Comment by tim-koehler** [comment]
Hey @pbnj,
thanks for letting me know. I am currently working on an update which fixes a few things. This is already one of them. The update will be published in the next days.

**Comment by tim-koehler** [comment]
fixed with v0.9.2
